### DISCUSSION TITLE
GRW团队招生啦！

### DISCUSSION THREAD

**Discussion by The-Astley** [share121/inter-knot#1984] [其他]
你好，陌生人
GRW团队招生了
无论你是跑酷大神
生存大师
还是PVP大佬
还是PVP大佬
又或是起床大佬
还是PVP大佬
还是PVP大佬
又或是起床大佬
又或是建筑大佬
还是PVP大佬
又或是建筑大佬
又或是建筑大佬
还是什么都不知道的小白
还是PVP大佬
又或是建筑大佬（放置绝世唐门）
又或是红石大佬
又或是红石大佬
还是PVP大佬
还是什么都不知道的小白
还是PVP大佬
还是PVP大佬
![872b906dfbc999d0](https://github.com/user-attachments/assets/10f76de5-7812-4880-94c7-0f36543af78d)

**Comment by lcshxl** [comment]
还是PVP大佬

**Comment by SENMAOJU** [comment]
#我是小白

**Comment by Dave-12138** [comment]
还是PVP大佬![流萤 没钱](https://bbs-static.miyoushe.com/static/2024/06/19/c638e43d0e2701e8f3e29a82dfd659fe_7613886348305980822.png)

**Comment by cpu644** [comment]
还是pvp大佬

**Comment by CloudSwordSage** [comment]
好耶！！我红石萌新！！只会***数电***的红石萌新！！！

**Comment by VacuolePaoo** [comment]
MeteorClient，启动！
> 把广告删了

**Comment by ghhccghk** [comment]
还是pvp大佬

**Comment by YuiJMQ** [comment]
还是尸V尸大佬

**Comment by reversed-PAI** [comment]
还是PVP大佬
